website/docs/software/plugins/external-notifications.md

```diff
@@ -0,0 +1,94 @@
+---
+id: ext-notif-plugin
```

Comment:
I'm happy to do this if that's the consensus, I just feel it looks nicer with links having dashes rather than underscores. ie docs/getting_started compared to /plugins/store-forward-plugin

Let me know!

Reply:
To be honest, I prefer the dashes in a url too. I just noticed it done otherwise and changed mine. If we decide that's why we like we can change things throughout to be uniform. I'm in favor of dashes. @sachaw?

Reply:
Agreed, I'll submit a pr with hyphens only soon